src/components/app/MenuButtons/index.js

```diff
@@ -165,7 +194,7 @@ class MenuButtonsImpl extends React.PureComponent<Props> {
     return (
       <button
         type="button"
-        className="menuButtonsButton menuButtonsRevertButton"
```

Comment:
Suggestion: For what it's worth, I would prefer to retain the perceived ordering of the buttons here. The "revert" button feels out of place here. This could be the case of how it was before, but right now with the new UI, it seems to stand out more to me.

### Before suggestion:

<img width="613" alt="image" src="https://user-images.githubusercontent.com/1588648/98595884-38e97a80-229c-11eb-9377-2b480f47ede8.png">


### After suggestion:

<img width="584" alt="image" src="https://user-images.githubusercontent.com/1588648/98595932-4999f080-229c-11eb-93fe-ede4087bdf41.png">

### Reload:

(notice how the ordering appears preserved)
<img width="471" alt="image" src="https://user-images.githubusercontent.com/1588648/98595988-646c6500-229c-11eb-85d2-6fe1db1dfcfb.png">

Reply:
Yeah this makes sense, thanks for the feedback.